recipes/grapheme/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "grapheme" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/grapheme-{{ version }}.tar.gz
+  sha256: 44c2b9f21bbe77cfb05835fec230bd435954275267fea1858013b102f8603cca
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - grapheme
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/alvinlindstam/grapheme
+  summary: Unicode grapheme helpers
+  license: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'

```

Comment:
```suggestion
  license_file: LICENSE
```

Reply:
The [source file](https://github.com/Brainor/staged-recipes/blob/f5b5ed9169b3b576683e02a4b28d2359a79413bb/recipes/grapheme/meta.yaml#L9) `.tar.gz`(the actual [link](https://pypi.io/packages/source/g/grapheme/grapheme-0.6.0.tar.gz)) doesn't contain a license file, so I have to put the license file in `staged-recipes/recipes/grapheme/`[LICENSE](https://github.com/Brainor/staged-recipes/blob/graphmeme/recipes/grapheme/LICENSE), hence the [environ["RECIPE_DIR"]](https://github.com/Brainor/staged-recipes/blob/f5b5ed9169b3b576683e02a4b28d2359a79413bb/recipes/grapheme/meta.yaml#L36) in line 36.

Reply:
Oh, I see what you mean with this [link](https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually) :)